src/Microsoft.AspNet.Mvc.Razor/RazorView.cs

```diff
@@ -42,6 +42,9 @@ public class RazorView : IView
             IsPartial = isPartial;
         }
 
+        /// <inheritdoc />
+        public string Path => RazorPage.Path;
```

Comment:
nit: ick; can't we limit ourselves to properties that look like properties?

in this case, `RazorPage.Path` won't change during the lifetime of the `RazorView`.  so the lambda is an unnecessary indirection; just use a get-only auto-property.